projects/mac_ai/testing/prepare_llama_cpp.py

```
@@ -60,12 +61,12 @@ def retrieve(prefix):
 def prepare_test(base_work_dir, platform, cleanup=True):
     try:
         retrieve_latest_version(base_work_dir)
-    except Exception as e:
+    except RuntimeError as e:
         if not cleanup:
             raise e
```

Ruff (0.13.3)

Use `raise` without specifying exception name

Remove exception name

(TRY201)